src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
```

Comment:
add `max-width: 100%` to avoid problems with mobiles.

Reply:
done